internal/ethapi/celo_block_receipt.go

```diff
@@ -0,0 +1,60 @@
+package ethapi
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+// GetBlockReceipt returns "system calls" receipt for the block with the given block hash.
+func (s *BlockChainAPI) GetBlockReceipt(ctx context.Context, hash common.Hash) (map[string]interface{}, error) {
```

Comment:
Please add a test for this endpoint.

Reply:
Added an issue for this here - https://github.com/celo-org/op-geth/issues/156